src/Core/src/Platform/Android/ImageButtonExtensions.cs

```diff
@@ -40,6 +40,9 @@ public static void UpdateCornerRadius(this ShapeableImageView platformButton, IB
 				.SetBottomLeftCorner(CornerFamily.Rounded, radius)
 				.SetBottomRightCorner(CornerFamily.Rounded, radius)
 				.Build();
+
+			int contentPadding = (int)(radius / 2);
+			platformButton.SetContentPadding(contentPadding, contentPadding, contentPadding, contentPadding);
```

Comment:
The changes are impacting the test Issue18242Test clipping the image:
![image](https://github.com/dotnet/maui/assets/6755973/48b33951-6273-42f3-91eb-fa24a95d20eb)

Reply:
Hmmm I'm not sure what is wrong here and if it is a good thing or not